packages/api/src/getNetwork/getNetwork.ts

```diff
@@ -1,22 +1,25 @@
-import { GEM_WALLET, NetworkResponse, RequestNetworkMessage } from '@gemwallet/constants';
+import { GEM_WALLET, NetworkResponsePayload, RequestGetNetworkMessage } from '@gemwallet/constants';
 
 import { sendMessageToContentScript } from '../helpers/extensionMessaging';
 
-export const getNetwork = async () => {
+export const getNetwork = async (): Promise<NetworkResponsePayload> => {
```

Comment:
Please rename: `NetworkResponsePayload` by `GetNetworkResponsePayload` and export it within the API types.

Reply:
Done